src/components/Section/index.js

```diff
@@ -0,0 +1,31 @@
+import classnames from "classnames";
```

Comment:
For some reason on my install, React needs to be included here otherwise it errors out.